src/components/__tests__/FormLogin.cy.js

```diff
@@ -420,123 +422,129 @@ describe('<FormLogin>', () => {
-                cy.intercept('POST', apiRefreshUrl, {
-                  statusCode: httpSuccessfullStatus,
-                  body: refreshTokensResponse,
-                })
-                  .as('refreshTokens')
-                  .then(() => {
-                    // set time to when JWT should be expired + 1 second
-                    clock.tick(timeUntilExpiration);
-                    // intercepted function apiRefreshUrl should be called
-                    cy.wait('@refreshTokens')
-                      .its('response.statusCode')
-                      .should('eq', httpSuccessfullStatus)
-                      .then(() => {
-                        // new JWT
-                        expect(store.getAccessToken).to.equal(
-                          refreshTokensResponse.access,
-                        );
-                        // JWT not be expired
-                        expect(store.isJwtExpired()).to.equal(false);
-                        expect(store.getJwtExpiration).to.equal(
-                          fixtureTokenRefreshExpirationTime,
-                        );
-                      });
-                  });
-              },
-            );
+      cy.fixture('loginRegisterResponseChallengeInactive.json').then(
+        (loginResponse) => {
+          cy.get('@clock').then((clock) => {
+            const store = useLoginStore();
+            cy.wrap(store.login({ username, password })).then((response) => {
+              expect(response).to.deep.equal(loginResponse);
+              expect(store.getAccessToken).to.equal(loginResponse.access);
+              expect(store.getRefreshToken).to.equal(loginResponse.refresh);
+              expect(store.getUser).to.deep.equal(loginResponse.user);
+              expect(store.getJwtExpiration).to.equal(
+                fixtureTokenExpirationTimeSeconds,
+              );
+              expect(store.getTimeUntilExpiration()).to.equal(
+                timeUntilExpirationSeconds, // time until expiration in seconds
+              );
+              expect(store.isJwtExpired()).to.equal(false);
+              cy.fixture('refreshTokensResponse.json').then(
+                (refreshTokensResponse) => {
+                  // intercept refresh token API call
+                  const apiBaseUrl = getApiBaseUrlWithLang(
+                    null,
+                    apiBase,
+                    apiDefaultLang,
+                    i18n,
+                  );
+                  const apiRefreshUrl = `${apiBaseUrl}${urlApiRefresh}`;
+                  cy.intercept('POST', apiRefreshUrl, {
+                    statusCode: httpSuccessfullStatus,
+                    body: refreshTokensResponse,
+                  })
+                    .as('refreshTokens')
+                    .then(() => {
+                      // set time to when JWT should be expired + 1 second
+                      clock.tick(timeUntilExpiration);
```

Comment:
Comment say _set time when JWT should be expired plus **1 second**_, but in code you don't extend time by 1 second.

And see my comment bellow.

Reply:
Resolved, see the comment [here](https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/629#discussion_r1821101201).